### ISSUE TITLE
ffmpeg crashes with error code 1: Invalid data

### ISSUE THREAD

**Issue by udochrist** [hjdhjd/homebridge-unifi-protect#703]
**Describe The Problem:**
Live video no longer working (possibly after latest UniFI update)

**To Reproduce:**
Click on Camera-Preview. Picture gets blurred and does never recover.

**Logs:**

```
[3/22/2022, 9:51:55 AM] [Homebridge UI] Homebridge log truncated by admin.
[3/22/2022, 9:52:07 AM] [UniFi Protect] udm-pro [UDM-PRO] G3 Instant [UVC G3 Instant]: Streaming request from 192.168.1.211: 1280x720@30fps, 299 kbps. Using 1920x1080@30fps (High), 802 kbps.
[3/22/2022, 9:52:16 AM] [UniFi Protect] udm-pro [UDM-PRO] G3 Instant [UVC G3 Instant]: Stopped video streaming session.
[3/22/2022, 9:52:16 AM] [UniFi Protect] udm-pro [UDM-PRO] G3 Instant [UVC G3 Instant]: FFmpeg process ended unexpectedly with an exit code of 1.
[3/22/2022, 9:52:16 AM] [UniFi Protect] udm-pro [UDM-PRO] G3 Instant [UVC G3 Instant]: FFmpeg command line that errored out was: /homebridge/node_modules/homebridge-unifi-protect/node_modules/ffmpeg-for-homebridge/ffmpeg -hide_banner -probesize 2048 -max_delay 500000 -r 30 -rtsp_transport tcp -i rtsps://<ip:port/key>?enableSrtp -map 0:v -vcodec copy -payload_type 99 -ssrc 3626614 -f rtp -srtp_out_suite AES_CM_128_HMAC_SHA1_80 -srtp_out_params <params> srtp://<ip:port>?rtcpport=<port>&localrtcpport=<port>&pkt_size=564 -map 0:a -acodec libfdk_aac -profile:a aac_eld -flags +global_header -f null -ar 16k -b:a 24k -bufsize 48k -ac 1 -payload_type 110 -ssrc 9029349 -f rtp -srtp_out_suite AES_CM_128_HMAC_SHA1_80 -srtp_out_params <params>  srtp://<ip:port>?rtcpport=<port>&localrtcpport=<port>&pkt_size=188
[3/22/2022, 9:52:16 AM] [UniFi Protect] rtsps://<ip:port/key>?enableSrtp: Invalid data found when processing input

[3/22/2022, 9:52:16 AM] [UniFi Protect] Exiting normally, received signal 15.
```

**Homebridge Configuration:**

```json
        {
            "controllers": [
                {
                    "address": "<ip>",
                    "password": "<pass>",
                    "username": "<user>"
                }
            ],
            "platform": "UniFi Protect"
        }
```

**Screenshots:**
Home.app after clicking preview image
![Screenshot 2022-03-22 at 09 58 03](https://user-images.githubusercontent.com/869637/159443887-9a029a55-abd0-47f0-a8a0-6a74e4b08504.png)

**Environment:**

* **Homebridge Version:** 1.4.0; Running as Docker Image on Synology x64 DS3612xs.
* **Node Version:** 16.14.0
* **Homebridge-UniFi-Protect Plugin Version:** 5.5.0
* **UniFi Protect Version:** 1.21.3
* **UniFi Protect Controller Type:** UDM Pro
* **UniFi Protect Controller Firmware Version:** 1.12.13
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:** MacBook Air M1 / iPhone and iPads also affected. 
* **Operating System and OS Version:** iOS 15.4 / macOS 12.3

**Comment by hjdhjd** [comment]
This is a UniFi Protect issue, not a plugin one…see: https://github.com/hjdhjd/homebridge-unifi-protect/issues/701#issuecomment-1073307884

It’s been a bit of a bloody couple of updates from Ubiquiti I’m afraid.

**Comment by udochrist** [comment]
Fixed with UniFi protect 1.21.4
